webapp-django/crashstats/signature/static/signature/js/signature_tab_reports.js

```diff
@@ -115,5 +130,47 @@ SignatureReport.ReportsTab.prototype.buildUrl = function (params) {
 
     // Return the whole URL.
     return this.dataUrl + queryString;
+};
+
+SignatureReport.ReportsTab.prototype.onAjaxSuccess = function (contentElement, data) {
+    SignatureReport.Tab.prototype.onAjaxSuccess.apply(this, arguments);
+    var that = this;
+
+    $('.sort-header', contentElement).each(function() {
+        $(this).click(function(event) {
```

Comment:
Why not just `$('.sort-header', contentElement).on('click', function(event) {`?
Now you have to create an event handler inside each loop of `.each()`.